### PR TITLE
Add an endpoint for Activity Stream to consume export plan data

### PR DESCRIPTION
**Pull request by rafa-garcia**
CONTEXT: This PR introduces an API endpoint that exposes export plan data for Activity Stream to consume.

The data structure has been designed to match the sections with all of the question groups available in the [web forms](https://www.great.gov.uk/export-plan/).

The model includes an `answers_count` dynamic property which mirrors the logic of the progress indicator on the frontend. This property is reflected in the JSON payload, making it easy to filter by completion degree of export plans.

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-1018
- [x] Jira ticket has the correct status
- [x] A clear/description pull request messaged added

### Merging

- [x] This PR can be merged by reviewers

### CODE REVIEW

File: activitystream/filters.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import datetime
+
+from django_filters import CharFilter, FilterSet
+
+
+class ActivityStreamCompanyExportPlanFilter(FilterSet):
+    after = CharFilter(method='filter_after')
```
Comment (sebs-code) [inline activitystream/filters.py:7]: Might be good to have a doc string on these (high level) explaining what they do.

Comment (rafa-garcia) [reply activitystream/filters.py:7]: I agree that it's a good practice, however, we're dealing with the simplest, standard `django-filter` filtersets which is wholly documented.

File: activitystream/views.py [hunk 2]
```diff
@@ -316,25 +320,20 @@ def list(self, request):
         )
 
 
-class ActivityStreamExportPlanDataViewSet(BaseActivityStreamViewSet):
-    """View set to list export plan data for the activity stream"""
+class ActivityStreamExportPlanView(ListAPIView):
+    authentication_classes = (ActivityStreamAuthentication,)
+    permission_classes = ()
```
Comment (sebs-code) [inline activitystream/views.py:325]: Do we need an empty permission_classes tuple here? Won't it be open by default?

File: activitystream/filters.py (hunk 1 above)
Comment (rafa-garcia) [reply activitystream/filters.py:7]: Do you mean the `filter_after` method, though? I pulled it from an AS view and that's undocumented. I'll put up a comment

File: activitystream/views.py (hunk 2 above)
Comment (rafa-garcia) [reply activitystream/views.py]: I'm putting back `permission_classes` with an empty tuple because otherwise the responses are `403`

**Review by sebs-code — APPROVED** [review]
Thanks for response to queries. Looks good.

**Pull request merged**